extensions/2.0/Khronos/KHR_materials_transmission/README.md

```diff
@@ -0,0 +1,129 @@
+Absorption is usually defined as an amount of light at each frequency that is absorbed over a given distance through a medium (usually described by Beer’s Law). However, since this extension deals exclusively with infinitely thin surfaces, we can treat absorption as a constant. In fact, rather than absorbed light, we can talk about its inverse: transmitted light. The `baseColor` of the material serves this purpose as it already defines how the light that penetrates the surface is colored by the material. In this model, the transmitted light will be modulated by this color as it passes through.
+<figure>
+  <img src="./figures/ConstantTransmission.png"/>
+<figcaption><em>The baseColor of the material (yellow, in this example) is used to tint the light being transmitted.</em></figcaption>
+</figure>
+
+## Refractive Blurring
+Since the surface is considered to be infinitely thin, we will ignore macroscopic refraction caused by the orientation of the surface. However, microfacets on either side of the thin surface will cause light to be refracted in random directions, effectively blurring the transmitted light. That is, the roughness of the surface directly causes the transmitted light to become blurred. This microfacet lobe is exactly the same as the specular lobe except sampled along the line of sight through the surface.
+<figure>
+  <img src="./figures/Roughness.png"/>
+<figcaption><em>Refraction due to surface roughness.</em></figcaption>
+</figure>
+
+## Blending
+The glTF `blendMode` is used for alpha-as-coverage, NOT for physically-based transparency (i.e. this extension). If alpha-as-coverage is not being used, the blend mode of the material should be set to "OPAQUE" even though it is transparent. Again, it's helpful to think of alpha-as-coverage as whether the physical surface is there or not. `transmission` applies to the surface material that exists.
+Note that alpha-as-coverage can still be used along with transmission as shown in the example below.
+<figure>
+  <img src="./figures/TransmissionWithMask.png"/>
+<figcaption><em>Alpha coverage and optical transparency can be used at the same time so that some areas of a surface are transparent while others disappear entirely.</em></figcaption>
+</figure>
+
+## Transparent Metals
+Metals effectively absorb all refracted light (light that isn't reflected), preventing transmission.
+The metallic parameter of a glTF material effectively scales the `baseColor` of the material toward black while, at the same time scaling the F0 (reflectivity) value towards 1.0. This makes the material opaque for metallic values of 1.0 because transmitted light is attenuated out by absorption. Therefore, for a material with `metallicFactor=1.0`, the value of `transmissionFactor` doesn't matter.
+
+## Implementing Transmission ##
```

Comment:
Please make it clear that this is an implementation note:

> **Implementation Note:**

Reply:
This section is normative. The non-normative "Implementation Notes" are below this section.